dll/win32/msafd/misc/dllmain.c

```diff
@@ -1819,6 +1819,38 @@ WSPAccept(
         }
     }
 
+    /* AcceptSocket asynchronous events */
+    if (Socket->SharedData->NonBlocking)
+    {
+        /* The socket created by the accept function has the same
+         * properties as the listening socket used to accept it. */
+        if (Socket->SharedData->AsyncEvents)
+        {
+            Status = WSPAsyncSelect(AcceptSocket,
```

Comment:
These functions do not return a `NTSTATUS`, use an appropriate variable to be consistent.